magda-web-server/src/index.ts

```diff
@@ -245,11 +245,43 @@ const argv = yargs
         type: "string",
         default: "Australia/Sydney"
     })
+    .option("homePageUrl", {
+        describe: "an alternative home page url.",
+        type: "string"
+    })
```

Comment:
Is it possible to set the default homePageUrl here rather than writing out the default in each place the homePageUrl is used?

Reply:
👍  Good idea~ considering the local dev use case for web client, I made the default value resolved in "config.ts" in web-client module instead.